backend/src/main/java/datastorage/SimpleDataStore.java

```diff
@@ -109,4 +112,19 @@ public Map<String, Player> getPlayers() {
    */
   @Override
   public void parseFile(String fileJson) {}
+
+  /**
+   * Method that adds a player to the queue
+   * @param ID, player to be added to the queue
+   */
+  public Player addQueue(String ID) throws NoItemFoundException {
+    try {
+      Player player = this.getPlayer(ID);
+      this.playerQueue.add(player);
+      return player;
+    } catch (NoItemFoundException e) {
```

Comment:
Thanks for the exception here by the way. It helps keep everything consistent for the datastore.